src/main/java/com/glucoseguardian/webbackend/application/rest/NotificaRest.java

```diff
@@ -0,0 +1,174 @@
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio Notifica getByTutore.
+   */
+  @PostMapping(value = "/getByTutore", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> getNotificaByTutore(
+      @RequestBody TutoreDto input) throws Exception {
+    ResponseEntity<RisultatoDto> response;
+    try {
+      ListDto<NotificaDto> dto = getService().findByDottore(input.getCodiceFiscale());
+      response = new ResponseEntity<>(dto, HttpStatus.OK);
+    } catch (UserNotFoundException | AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      response = new ResponseEntity<>(new RisultatoDto("Errore durante la ricerca della notifica"),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio Notifica getByAdmin.
+   */
+  @PostMapping(value = "/getByAdmin", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> getNotificaByAdmin(
+      @RequestBody AdminDto input) throws Exception {
+    ResponseEntity<RisultatoDto> response;
+    try {
+      ListDto<NotificaDto> dto = getService().findByDottore(input.getCodiceFiscale());
+      response = new ResponseEntity<>(dto, HttpStatus.OK);
+    } catch (UserNotFoundException | AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      response = new ResponseEntity<>(new RisultatoDto("Errore durante la ricerca della notifica"),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+      ex.printStackTrace();
+      ex.printStackTrace();
+    }
+    return CompletableFuture.completedFuture(response);
+  }
+
+  /**
+   * Metodo che gestisce il servizio save Notifica.
+   */
+
+  @PostMapping(value = "/save", produces = MediaType.APPLICATION_JSON_VALUE)
+  public @ResponseBody CompletableFuture<ResponseEntity<RisultatoDto>> saveNotifica(
+      @RequestBody NotificaDto input) throws UserNotFoundException {
+
+    // TODO: Add custom checks (es. length, null etc..)
+
+    boolean result = false;
+    try {
+      result = getService().send(input);
+    } catch (AccessDeniedException ex) {
+      throw ex;
+    } catch (Exception ex) {
+      ex.printStackTrace();
+    }
+
+    if (result) {
+      return CompletableFuture.completedFuture(
+          new ResponseEntity<>(new RisultatoDto("Notifica inserito correttamente"), HttpStatus.OK));
+    } else {
+      return CompletableFuture.completedFuture(
+          new ResponseEntity<>(new RisultatoDto("Errore durante l'inserimento della Notifica"),
+              HttpStatus.INTERNAL_SERVER_ERROR));
+    }
+  }
+
```

Comment:
Missing updateStato endpoint. This method is missing in Service too.. So it should be added there first.